tests/Makefile

```diff
@@ -62,7 +62,7 @@ build: framework
 run: build
 	@echo -e ${GREEN}Running [$@]${NC}
 	@mkdir -p ./tests/logs
-	@bash -c "cd ./bin;./ut-test -bp ./../logs"
+	@bash -c "cd ./bin;./ut-test -bp ./assets/test_kvp.yaml -f ../logs/"
```

Comment:
update to use -l instead, then by default we stop filling /tmp

Reply:
I think this was because originally automated mode didn't generate logs or something so we can ignore that..  But I think yo  will need to call setFilename to set the filename into the subdirectory but it doesn't need to be a switch, and you can move that logic out of the main file, and put it in setFolder function in ut.